core/templates/dev/head/pages/Base.ts

```diff
@@ -12,22 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+require('domain/utilities/UrlInterpolationService.ts');
+
 /**
  * @fileoverview Oppia's base controller.
  */
 
 oppia.controller('Base', [
   '$document', '$rootScope', '$scope', 'AlertsService', 'BackgroundMaskService',
-  'SidebarStatusService', 'UrlService', 'DEV_MODE', 'SITE_FEEDBACK_FORM_URL',
-  'SITE_NAME',
+  'SidebarStatusService', 'UrlInterpolationService', 'UrlService', 'DEV_MODE',
+  'SITE_FEEDBACK_FORM_URL', 'SITE_NAME',
   function($document, $rootScope, $scope, AlertsService, BackgroundMaskService,
-      SidebarStatusService, UrlService, DEV_MODE, SITE_FEEDBACK_FORM_URL,
-      SITE_NAME) {
+      SidebarStatusService, UrlInterpolationService, UrlService, DEV_MODE,
+      SITE_FEEDBACK_FORM_URL, SITE_NAME) {
```

Comment:
Please add requires for all the services we use in this controller (I'm not sure why aren't they added).

Reply:
Done